src/service/impl/trips/converters.ts

```diff
@@ -155,6 +156,7 @@ function mapToLegacyLine(line: Line_v2): Line | undefined {
   if (!line) return;
   return {
     ...line,
+    flexibleLineType: line.flexibleLineType as FlexibleLineType,
```

Comment:
Not sure about this I had undefined from before but I saw that FlexibleLineType is part of V1

Reply:
`@entur/sdk` is deprecated, so rather use the type from journey planner types v3. If that is `string`, so be it.

Reply:
@gorandalum V2 requires a String but Line which I think is V1 requires the type so that is why I used the deprecated FlexibleLineType

Reply:
We had a discussion and we decided to use undefined instead for legacy.